SCF/SCF-3/GeneralCalculator.java

```diff
@@ -0,0 +1,12 @@
+/*
+ *  Generalized Calculator
+ */
+public interface GeneralCalculator {
+	String addHexaNum(String firstNumber, String secondNumber);

```

Comment:
Need to add one more Interface Convert which will be implemented by Conversion class. This Interface will be used to do the conversion for different bases(16, 8, 2, etc). (High priority)